sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/StatsbeatHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.ConnectionString;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class StatsbeatHelper
+    {
+        internal static void InitializeStatsbeat(string connectionString)
```

Comment:
who calls this?

Reply:
This will need to be added to all 3 signals.